test/utilities/cliCommands.ts

```diff
@@ -0,0 +1,217 @@
+      `org list failed with exit code ${sfOrgListResult.exitCode}\n stderr ${sfOrgListResult.stderr}`
+    );
+  }
+  debug(`orgList results ${JSON.stringify(sfOrgListResult)}`);
+  return sfOrgListResult;
+}
+
+export async function scratchOrgCreate(
+  edition: string,
+  definitionFile: string,
+  scratchOrgAliasName: string,
+  durationDays: number
+): Promise<SfCommandRunResults> {
+  log('calling "sf org:create:scratch"...');
+  const args = [
+    '--edition',
+    edition,
+    '--definition-file',
+    definitionFile,
+    '--alias',
+    scratchOrgAliasName,
+    '--duration-days',
+    durationDays.toString(),
+    '--set-default',
+    '--json'
+  ];
+
+  const sfOrgCreateResult = await runCliCommand('org:create:scratch', ...args);
+
+  if (sfOrgCreateResult.exitCode) {
+    log(
+      `create scratch org failed. Exit code: ${sfOrgCreateResult.exitCode}. \ncreate scratch org failed. Raw stderr: ${sfOrgCreateResult.stderr}`
+    );
+    throw new Error(sfOrgCreateResult.stderr);
+  }
+
+  log(`..."sf org:create:scratch" finished`);
+  debug(`scratchOrgCreate results ${JSON.stringify(sfOrgCreateResult)}`);
+
+  return sfOrgCreateResult;
+}
+
+export async function setAlias(
+  devHubAliasName: string,
+  devHubUserName: string
+): Promise<SfCommandRunResults> {
+  const setAliasResult = await runCliCommand('alias:set', `${devHubAliasName}=${devHubUserName}`);
+  if (setAliasResult.exitCode) {
+    log(
+      `alias failed. Exit code: ${setAliasResult.exitCode}. \nRaw stderr: ${setAliasResult.stderr}`
+    );
+    throw new Error(setAliasResult.stderr);
+  }
+  return setAliasResult;
+}
+
+export async function installJestUTToolsForLwc(projectFolder: string | undefined) {
+  if (!projectFolder) {
+    throw new Error('cannot setup lwc tests without a project folder.');
+  }
+
+  const jestInstallResult = await runCliCommand('force:lightning:lwc:test:setup', {
+    cwd: projectFolder
+  });
+
+  if (jestInstallResult.exitCode) {
+    log(
+      `alias failed. Exit code: ${jestInstallResult.exitCode}. \nRaw stderr: ${jestInstallResult.stderr}`
+    );
+    throw new Error(jestInstallResult.stderr);
+  }
+  return jestInstallResult;
+}
+
+export async function createUser(
+  systemAdminUserDefPath: string,
+  targetOrg: string | undefined
+): Promise<SfCommandRunResults> {
+  if (!targetOrg) {
+    throw new Error('cannot create user with target');
+  }
+  const sfOrgCreateUserResult = await runCliCommand(
+    'org:create:user',
+    '--definition-file',
+    systemAdminUserDefPath,
+    '--target-org',
+    targetOrg
+  );
+  if (sfOrgCreateUserResult.exitCode) {
+    log(
+      `org crate user failed Exit code: ${sfOrgCreateUserResult.exitCode}. \nRaw stderr: ${sfOrgCreateUserResult.stderr}`
+    );
+    throw new Error(sfOrgCreateUserResult.stderr);
+  }
+  debug(`createUser results ${JSON.stringify(sfOrgCreateUserResult)}`);
+  return sfOrgCreateUserResult;
+}
+
+export function removedEscapedCharacters(result: string): string {
```

Comment:
nit - change function name to present tense

```suggestion
export function removeEscapedCharacters(result: string): string {
```